### ISSUE TITLE
Multiple Namelists in CLI

### ISSUE THREAD

**Issue by kgallmei** [marshallward/f90nml#156]
I would very much appreciate, if one could change variables in multiple namelists via the cli simultanuously.

So instead of giving '-g NML' and '-v "VAR=val"', which works only for only one 'NML' (I do not know, for which one, if one gives multiple '-g ...'), one could maybe implement an argument parser, which uses the syntax "-v "NML:VAR=val" and throw the '-g ...'?

Thanks for the cool work anyway.

**Comment by marshallward** [comment]
I agree, the current flags are neither good nor easy to use.  I believe could use an entire overhaul.

Your suggestion seems good to me, I'll also look at how `jq` does this sort of thing.
